feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go

```diff
@@ -596,25 +622,47 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		gnmi.Replace(t, dut, gnmi.OC().Qos().Config(), q)
 	}
 
-	t.Logf("Create qos queue management profile config")
-	ecnConfig := struct {
-		profileName  string
-		ecnEnabled   bool
-		minThreshold uint64
-	}{
-		profileName:  "ECNProfile",
-		ecnEnabled:   true,
-		minThreshold: uint64(80000),
+	if deviations.EcnProfileRequiredDefinition(ondatra.DUT(t, "dut")) {
```

Comment:
You already have a reference to the DUT, so just `if deviations.EcnProfileRequiredDefinition(dut)`